### ISSUE TITLE
Is This library still maintained ?

### ISSUE THREAD

**Issue by GulajavaMinistudio** [stephanenicolas/robospice#456]
Is this library still maintained ?

**Comment by farmazon3000** [comment]
Last commit is from 2014 so go figure...

**Comment by mykolaj** [comment]
What difference does the date of a last commit make?
The library is doing its functions flawlessly. I don't think that `android.os.Service` implementation will change any time soon. So RS possibly will be useful for years to come without any commits.

**Comment by softwaremaverick** [comment]
Yeah I'll pop into the code at some point to see what commits can be merged. I just don't need anything additional from it right now.

On 4 May 2016 14:35:01 BST, Tony notifications@github.com wrote:

> What difference does the date of a last commit make?
> The library is doing its functions flawlessly. I don't think that
> `android.os.Service` implementation will change any time soon. So RS
> possibly will be useful for years to come without any commits.
> 
> ---
> 
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/stephanenicolas/robospice/issues/456#issuecomment-216866582

## 

Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by deinlandel** [comment]
> The library is doing its functions flawlessly. 

It does have a couple of bugs, though. Just look at the issues list. So even if library is feature-complete, it still needs maintenance.

**Comment by stephanenicolas** [comment]
The core maintainers doesn't have time to maintain them. At least I don't
and it does look like I am not the only one in this case.

If anyone wants to fix a bug, please ping me on my personal email address
to merge your PRs and ship a new version. We would appreciate new
volunteers.

S.

2016-05-22 22:08 GMT-07:00 Lev Shock notifications@github.com:
